plugins/uefi-dbx/fu-uefi-dbx-device.c

```diff
@@ -144,6 +144,31 @@ fu_uefi_dbx_device_probe(FuDevice *device, GError **error)
 	return fu_uefi_dbx_device_set_version_number(device, error);
 }
 
+static void
+fu_uefi_dbx_device_report_metadata_pre(FuDevice *device, GHashTable *metadata)
+{
+	guint64 filesystem_used;
+	g_autoptr(GError) error_local = NULL;
+	g_autoptr(GFile) file = g_file_new_for_path("/sys/firmware/efi/efivars");
+	g_autoptr(GFileInfo) info = NULL;
+
+	info = g_file_query_info(file,
+				 G_FILE_ATTRIBUTE_FILESYSTEM_USED,
+				 G_FILE_QUERY_INFO_NONE,
+				 NULL,
+				 &error_local);
+	if (info == NULL) {
+		g_debug("failed to get efivar used space: %s", error_local->message);
+		return;
+	}
+	filesystem_used = g_file_info_get_attribute_uint64(info, G_FILE_ATTRIBUTE_FILESYSTEM_USED);
+	if (filesystem_used > 0) {
```

Comment:
Hmm, I was really hoping we could have a case that we know patch is here or patch isn't.  I think it's best to record even the zero.  As the patch lands, the reports should hopefully start showing people flip from 0 to non-zero, and then we can know it's working.

Reply:
Makes sense to me. Fixed.